datagateway_api/src/datagateway_api/database/models.py

```diff
@@ -671,6 +974,70 @@ class INVESTIGATION(Base, EntityHelper, metaclass=EntityMeta):
     )
 
 
+class INVESTIGATIONFACILITYCYCLE(Base, EntityHelper, metaclass=EntityMeta):
+    __tablename__ = "INVESTIGATIONFACILITYCYCLE"
+    __singularfieldname__ = "investigationFacilityCycle"
+    __pluralfieldname__ = "investigationFacilityCycles"
+    __table_args__ = (
+        Index(
+            "UNQ_INVESTIGATIONFACILITYCYCLE_0", "FACILITYCYCLE_ID", "INVESTIGATION_ID",
+        ),
+    )
+
+    id = Column("ID", BigInteger, primary_key=True)
+    createId = Column("CREATE_ID", String(255), nullable=False)
+    createTime = Column("CREATE_TIME", DateTime, nullable=False)
+    modId = Column("MOD_ID", String(255), nullable=False)
+    modTime = Column("MOD_TIME", DateTime, nullable=False)
+    facilityCycleId = Column(
+        "FACILITYCYCLE_ID", ForeignKey("FACILITYCYCLE.ID"), nullable=False,
+    )
+    investigationID = Column(
+        "INVESTIGATION_ID", ForeignKey("INVESTIGATION.ID"), nullable=False, index=True,
+    )
+
+    FACILITYCYCLE = relationship(
+        "FACILITYCYCLE",
+        primaryjoin="INVESTIGATIONFACILITYCYCLE.facilityCycleId == FACILITYCYCLE.id",
+        backref="investigationFacilityCycles",
+    )
+    INVESTIGATION = relationship(
+        "INVESTIGATION",
+        primaryjoin="INVESTIGATIONFACILITYCYCLE.investigationID == INVESTIGATION.id",
+        backref="investigationFacilityCycles",
+    )
+
+
+class INVESTIGATIONFUNDING(Base, EntityHelper, metaclass=EntityMeta):
+    __tablename__ = "INVESTIGATIONFUNDING"
+    __singularfieldname__ = "investigationFunding"
+    __pluralfieldname__ = "investigationFundings"
+    __table_args__ = (
+        Index("UNQ_INVESTIGATIONFUNDING_0", "FUNDING_ID", "INVESTIGATION_ID"),
+    )
+
+    id = Column("ID", BigInteger, primary_key=True)
+    createId = Column("CREATE_ID", String(255), nullable=False)
+    createTime = Column("CREATE_TIME", DateTime, nullable=False)
+    modId = Column("MOD_ID", String(255), nullable=False)
+    modTime = Column("MOD_TIME", DateTime, nullable=False)
+    fundingId = Column("FUNDING_ID", ForeignKey("FUNDINGREFERENCE.ID"), nullable=False)
+    investigationID = Column(
+        "INVESTIGATION_ID", ForeignKey("INVESTIGATION.ID"), nullable=False, index=True,
+    )
+
+    FUNDINGREFERENCE = relationship(
+        "FUNDINGREFERENCE",
+        primaryjoin="INVESTIGATIONFUNDING.fundingId == FUNDINGREFERENCE.id",
+        backref="investigationFundings",
```

Comment:
The correct back reference is `investigations`. Please could this be changed?

Reply:
Yes, changed